src/main/kotlin/de/livepoll/api/config/SecurityConfig.kt

```diff
@@ -27,6 +27,7 @@ class SecurityConfig(
             .antMatchers("/v1/account/register").permitAll()
             .antMatchers("/v1/account/confirm").permitAll()
             .antMatchers("/v1/account/login").permitAll()
+            .antMatchers("/v1/polls/{id:[\\d]+}").permitAll()
```

Comment:
Does this really only allow GET requests? Seems like all other request types are also permitted?!

Reply:
I don't know exactly, but we want to change it again anyway. In addition, all other endpoints require an authentication user who is missing and that would throw an exception if you send a request

Reply:
Fine for now, but let's change this after Wednesday

Reply:
Okay